site/pages/create-consequence-operator/[disruptionId]/[consequenceIndex].page.tsx

```diff
@@ -193,7 +193,11 @@ const CreateConsequenceOperator = (props: CreateConsequenceOperatorProps): React
                         <input
                             type="hidden"
                             name="consequenceOperators"
-                            value={JSON.stringify(pageState.inputs.consequenceOperators)}
+                            value={
```

Comment:
This could be

```ts
JSON.stringify(pageState.inputs.consequenceOperators || [])
```

to make it a bit simpler

Reply:
done